test/jest-e2e.json

```diff
@@ -1,10 +1,12 @@
 {
   "moduleFileExtensions": ["js", "json", "ts"],
-  "rootDir": ".",
-  "roots": ["../src"],
+  "rootDir": "../",
```

Comment:
This file didn't move so what is the reason for the reference to the `rootDir` changing?

Reply:
After several tests, I concluded `rootDir` has to be set to the root folder in order to correctly generate the coverage report (otherwise Jest generates an empty report). I had to change it because before this change coverage wasn't generated, so the tests ran correctly, but it seems it has to be changed to generate the coverage.

Reply:
Interesting! Thanks for the explanation!